.travis.yml

```diff
@@ -34,9 +34,11 @@ jobs:
       install:
         - npm ci
       script:
-        - npm run lint
-        - npm run check-local-changes
-        - npm run test-unit -- --ci --maxWorkers=2 --cacheDirectory="$HOME/.jest-cache"
+        - $( npm bin )/concurrently --kill-others-on-fail \
+          "npm run lint" \
+          "npm run check-local-changes" \
+          "npm run check-licenses" \
+          "npm run test-unit -- --ci --maxWorkers=2 --cacheDirectory=\"$HOME/.jest-cache\""
```

Comment:
I'd be interested in seeing what putting `npm run lint`, `npm run check-local-changes`, and `npm run check-licenses` into their own job adds to the build times.  I don't like the idea of doing concurrently here because it means for someone relying on travis has no insight into whether the subsequent tasks would pass or not.

Reply:
> I'd be interested in seeing what putting `npm run lint`, `npm run check-local-changes`, and `npm run check-licenses` into their own job adds to the build times.

Yeah, this is what prompted [my most recent thread comments](https://wordpress.slack.com/archives/C02QB2JS7/p1551887679164000?thread_ts=1551275461.296700&cid=C02QB2JS7) in Slack. I agree that ideally they'd be defined as separate jobs, but it's not clear how to do that without increasing build times (kinda contrary to the point of the pull request). 

On the general topic of "making sense of the build output", there might be some other options to explore for organizing the results in a way which is more easy to read. Right now, the concurrent output is a bit of a mess, since it just spews everything to `stdout` in the order it's received (i.e. parallel scripts each with their own outputs are intermixed). I think it comes down to: Can we leverage / find new / develop the tool to organize the output in a more readable fashion?

> I don't like the idea of doing concurrently here because it means for someone relying on travis has no insight into whether the subsequent tasks would pass or not.

I guess it depends if it's more in the interest of the developer to have faster build times, or a more thorough report of every issue in the build if there are multiple issues. I don't really have a strong leaning one way or the other, but I'd be inclined to think the former would benefit more people in the vast majority of cases.

Reply:
Have you tested with `--maxWorkers=4`? Travis VMs have two CPUs available, so it's usually a good idea to have more workers than that, so the CPUs are fully utilised: the workers will likely have I/O wait time to deal with.